readme.md

```diff
@@ -30,6 +30,11 @@ http: {
 * Indentation based syntax
 * Annotations for providing additional information
 
+The top level of a `.bru` file is an implicit Multimap. It does not require
+or support braces.
+
+Except where noted, blank lines and whitespace are ignored.
```

Comment:
This is related to the comment above about whitespace indentation, but indicates that although there is whitespace, it is *not* included except where noted.

Reply:
👍